nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/internal/memory/CloseValidationMemoryMgr.java

```diff
@@ -70,7 +70,7 @@ public void release(INDArray array) {
         if (released.get(array)) {
             //Already released
             InferenceSession is = sd.getSessions().get(Thread.currentThread().getId());
-            IdentityDependencyTracker<INDArray, InferenceSession.Dep> arrayUseTracker = is.getArrayUseTracker();
+            AbstractDependencyTracker<INDArray, InferenceSession.Dep> arrayUseTracker = is.getArrayUseTracker();
```

Comment:
Are there any reasons why we can't just use the interface DependencyTracker instead?

Reply:
we don't have interface